docs/rootstock/setup_regtest.md

```diff
@@ -0,0 +1,136 @@
+  </svg>
+</div>
+  <a href="https://optimism.io"><img alt="Optimism" src="https://raw.githubusercontent.com/ethereum-optimism/brand-kit/main/assets/svg/OPTIMISM-R.svg" width=600></a>
+  <br />
+  <h3><a href="https://optimism.io">Optimism</a> is <del>Ethereum</del> <ins>Rootstock</ins>, scaled.</h3>
+  <br />
+</div>
+
+# Guide to setup local L1 and L2 development nodes
+
+You will need **5 TTYs** for running the both **L1** and **L2** nodes.
+For the purpose of this guide we will assume:
+
+```shell
+export PATH_RSKSMART="$HOME/src/rsksmart/"
+```
+
+This is illustrative, but by all means, set it in each TTY to copy-paste from this guide and/or set it to your desired location.
+
+#### TTY 1
+
+1. clone op-geth to serve as the sequencer client and build it
+
+    ```shell
+    cd $PATH_RSKSMART  # if not there already
+    git clone git@github.com:rsksmart/op-geth.git op-sequencer
+    cd op-sequencer
+    git swich rsk/initial_attempt
```

Comment:
We may need to change it to point to rsk/poc-v0

Reply:
done